src/graphql-global-id/src/GlobalID.js

```diff
@@ -32,6 +32,10 @@ export function fromGlobalId(opaqueID: string): string {
   return unbasedGlobalID.substring(delimiterPos + 1);
 }
 
+export function toGlobalId(type: string, id: string | number): string {
```

Comment:
Could you please explain why we need to export this function?

Reply:
I mean, what is the use case?

Reply:
I want to add `nodeId` into our current REST API so I can call GraphQL query that needs the global ID.

Reply:
Aha, I see 🙂